fixtures/laserworld/el-400rgb-mk2.json

```diff
@@ -1,135 +1,149 @@
+          "effectName": "Static pattern"
         },
         {
-          "dmxRange": [254, 255],
+          "dmxRange": [200, 255],
           "type": "Effect",
-          "effectName": "Dynamische Muster"
+          "effectName": "Dynamic pattern"
         }
       ]
     },
-    "Musterauswahl": {
+    "Pattern Selection": {
       "capability": {
         "type": "Effect",
-        "effectName": "Musterauswahl"
+        "effectName": "Pattern Selection"
       }
     },
-    "X-Achse": {
+    "Horizontal Angle": {
       "capabilities": [
         {
           "dmxRange": [0, 10],
-          "type": "Effect",
-          "effectName": "Mitte"
+          "type": "BeamPosition",
+          "horizontalAngle": "center"
         },
         {
           "dmxRange": [11, 255],
-          "type": "Effect",
-          "effectName": "Position"
+          "type": "BeamPosition",
+          "horizontalAngleStart": "left",
+          "horizontalAngleEnd": "right",
+          "helpWanted": "Are the horizontal angle start and end values correct?"
         }
       ]
     },
-    "Y-Achse": {
+    "Vertical Angle": {
       "capabilities": [
         {
           "dmxRange": [0, 10],
-          "type": "Effect",
-          "effectName": "Y-Achse"
+          "type": "BeamPosition",
+          "verticalAngle": "center"
         },
         {
           "dmxRange": [11, 255],
-          "type": "Effect",
-          "effectName": "Position"
+          "type": "BeamPosition",
+          "verticalAngleStart": "bottom",
+          "verticalAngleEnd": "top",
+          "helpWanted": "Are the vertical angle start and end values correct?"
         }
       ]
     },
-    "Scan Geschwindigkeit": {
+    "Scanning Speed": {
       "capability": {
-        "type": "Effect",
-        "effectName": "Scan Geschwindigkeit"
+        "type": "Speed",
+        "speedStart": "fast",
+        "speedEnd": "slow",
+        "helpWanted": "Are the scanning speed start and end values correct?"
       }
     },
-    "Muster Geschwindigkeit": {
+    "Dynamic Pattern Speed": {
       "capability": {
-        "type": "Effect",
-        "effectName": "Muster Geschwindigkeit"
+        "type": "EffectSpeed",
+        "speedStart": "fast",
+        "speedEnd": "slow",
+        "helpWanted": "Are the dynamic pattern speed start and end values correct?"
       }
     },
     "Zoom": {
       "capability": {
         "type": "Zoom",
         "angleStart": "narrow",
-        "angleEnd": "wide"
+        "angleEnd": "wide",
+        "helpWanted": "Are the zoom start and end values correct?"
       }
     },
-    "Colour": {
+    "Color": {
       "capability": {
         "type": "Effect",
-        "effectName": "Colour"
+        "effectName": "Color"
       }
     },
-    "Colour Segment": {
+    "Color Segment": {
       "capability": {
         "type": "Effect",
-        "effectName": "Colour Segment"
+        "effectName": "Color Segment"
       }
     }
```

Comment:
I think this should rather be a `Generic` capability rather than an Effect. Do you know what it does? Can we also add a `helpWanted` message here otherwise?

Reply:
Unfortunately not. Gonna go for the `helpWanted` message.